notebooks/212-onnx-style-transfer/README.md

```diff
@@ -0,0 +1,23 @@
+# Neural Style Transfer with ONNX models and OpenVINO
+
+![Neural Style Transfer Network Output](https://user-images.githubusercontent.com/77325899/147366340-4a281b0b-066d-4114-b9ef-48634d733095.png)
+
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F212-onnx-style-transfer%2F212-onnx-style-transfer.ipynb)
+
+This notebook demonstrates [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) with ONNX models and OpenVINO. Neural style transfer networks take two images as input during training: a source image, and a style reference image. The network transforms the source image and returns an image with the style of the reference image applied to the source image.
```

Comment:
```suggestion
This notebook demonstrates [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) with ONNX models and OpenVINO. Neural style transfer networks take two images as input during inference: a source image, and a style reference image. The network transforms the source image and returns an image with the style of the reference image applied to the source image.
```
Should this be inference (not training)?

Reply:
No (see comment above for the same sentence in the notebook).